corehq/apps/export/models/new.py

```diff
@@ -382,13 +408,14 @@ def _generate_schema_from_case_property_mapping(case_property_mapping, app_versi
         for case_type, case_properties in case_property_mapping.iteritems():
             group_schema = ExportGroupSchema(
                 path=[case_type],
-                last_occurrence=app_version,
+                last_occurrences={app_id: app_version},
             )
             for prop in case_properties:
                 group_schema.items.append(ScalarItem(
                     path=[prop],
                     label=prop,
-                    last_occurrence=app_version,
+                    last_occurrences={app_id: app_version},
+                    app_id=app_id,
```

Comment:
It doesn't look like `ScalarItem` does anything with with the `app_id` argument. (also occurs on lines 441 and 451)